requirements.txt

```diff
@@ -83,3 +83,70 @@ netaddr==0.8.0
 vulners==2.0.10
 fontawesomefree==6.4.0
 azure-devops==7.1.0b3
+
+
+#New Dependencies
+azure-core==1.26.4
+MarkupSafe==2.1.2
+prompt-toolkit==3.0.38
+billiard==3.6.4.0
+inflection==0.5.1
+requests-toolbelt==0.10.1
+smmap==4.0.0
+wcwidth==0.2.5
+#pyOpenSSL==21.0.0
+kombu==5.2.3
+python-jose==3.3.0
+six==1.16.0
+attrs==21.2.0
+idna==2.10
+packaging==21.0
+pysaml2==7.0.1
+pyrsistent==0.18.0
+click-repl==0.2.0
+pycparser==2.20
+et-xmlfile==1.1.0
+google-auth-oauthlib==0.4.6
+coreschema==0.0.4
+multidict==5.2.0
+pyasn1==0.4.8
+ecdsa==0.17.0
+prometheus-client==0.11.0
+Deprecated==1.2.13
+click-didyoumean==0.3.0
+webencodings==0.5.1
+uritemplate==3.0.1
+pilkit==2.0
+yarl==1.7.0
+django-appconf==1.0.5
+click==8.0.3
+jsonschema==4.1.1
+rsa==4.7.2
+python3-openid==3.2.0
+certifi==2021.10.8
+PyNaCl==1.4.0
+charset-normalizer==2.0.7
+xmlschema==1.8.0
+sqlparse==0.4.2
+cffi==1.15.0
+gitdb==4.0.7
+Jinja2==3.0.2
+PyYAML==6.0
+click-plugins==1.1.1
+ruamel.yaml.clib==0.2.6
+asgiref==3.5.2
+ruamel.yaml==0.17.16
+itypes==1.2.0
+elementpath==2.3.2
+requests-oauthlib==1.3.0
+vine==5.0.0
+wrapt==1.13.2
+appdirs==1.4.4
+amqp==5.0.9
+coreapi==2.3.3
+msrest==0.7.1
+tinycss2==1.1.1
+typing-extensions==4.6.3
```

Comment:
because this dependency ?

Reply:
The dependency is requested in some XML files